yb-voyager/src/tgtdb/oracle.go

```diff
@@ -766,3 +766,52 @@ func (tdb *TargetOracleDB) GetTotalNumOfEventsImportedByType(migrationUUID uuid.
 func (tdb *TargetOracleDB) MaxBatchSizeInBytes() int64 {
 	return 2 * 1024 * 1024 * 1024 // 2GB
 }
+
```

Comment:
Same comments that were provided on `yugabytedb.go` should also be applicable here.